ethkey/src/secret.rs

```diff
@@ -48,6 +48,11 @@ impl Secret {
 		Secret { inner: h }
 	}
 
+	/// Creates zero key, which is invalid for crypto operations, but valid for math operation.
+	pub fn zero() -> Self {
+		Secret { inner: Default::default() }
```

Comment:
Didn't wanted to implement `Default` for `Secret`, since it actually produces invalid `Secret`.